packages/core/src/ui/components/NamiMigration/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { sx, Flex, Text, Box, Button } from '@lace/ui';
+import { ReactComponent as CloseComponent } from '@lace/icons/dist/CloseComponent';
+import { ReactComponent as CheckComponent } from '@lace/icons/dist/CheckComponent';
+import NamiImg from './nami.png';
+import LaceImg from './lace.png';
+import ArrowImg from './arrow-right.png';
+import { WalletImg } from './WalletImg';
+import { Wizard } from '../Wizard';
+
+interface Props {
+  termsOfServiceUrl: string;
+  privacyPolicyUrl: string;
+  faqUrl: string;
+}
+
+export const Welcome = ({ termsOfServiceUrl, privacyPolicyUrl, faqUrl }: Props): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <Wizard step="welcome">
+      <Box mb="$32">
+        <Text.SubHeading
```

Comment:
We now have a color prop. Maybe this will work:
```suggestion
        <Text.SubHeading color="primary"
```

Reply:
@lucas-barros updated!